account_bank_statement_import_paypal/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,12 @@
+In the menu Accounting > Configuration > Accounting > Bank Accounts,
+make sure that you have your Paypal bank account with the following parameters:
+* Bank Account Type: Normal Bank Account
+* Account Number: the email address associated with your Paypal account
+* Account Journal: the journal associated to your Paypal account
+
+TIPS
+----
+For now only French and English report are supported.
```

Comment:
@cubells Could we have them in spanish as well?

Reply:
That's out of the scope of the module right now. This is for the OCA instance